tests/testthat/test-test_basic_finalsize.R

```diff
@@ -0,0 +1,23 @@
+test_that("Check basic final size calculation works", {
```

Comment:
It would be very useful to test that errors / warnings / messages (currently, I think only errors) do happen when they are supposed to, with the correct message. See [this ](https://github.com/epiverse-trace/linelist/blob/main/tests/testthat/test-make_linelist.R)for an example.

Reply:
Added as #24, addressed in https://github.com/epiverse-trace/finalsize/commit/c9abe30ee6cacf8060c98d7163206f894d046f5d